### ISSUE TITLE
Setup

### ISSUE THREAD

**Issue by sahilahemad05** [rakeshgowdan/Online-Job-Portal-Java#1]
hey sir, would you please help me to setup this project in my laptop please

**Comment by Mayur047** [comment]
Yes please, I tried a lot for setting the project but i am getting error somehow
